dnsupdate.sh

```diff
@@ -2,14 +2,25 @@
 
 USERNAME="" #INWX USERNAME
 PASSWORD="" #INWX PASSWORD
-DNSID="" #DNS Entry ID
+DNSIDv4="" #DNS Entry ID
+DNSIDv6="" #DNS Entry ID
 APIHOST="https://api.domrobot.com/xmlrpc/"
-OLDIP=$(cat old.ip)
-NEWIP=$(curl -s curlmyip.com)
+OLDIPv4=$(cat old.ipv4)
+OLDIPv6=$(cat old.ipv6)
+NEWIPv4=$(curl -s curlmyip.com)
+NEWIPv6=$(curl -s curlmyip6.com)
```

Comment:
What exactly do you mean? How do I know that there is no IPv6 before `curl -s curlmyip6.com`?